### PR TITLE
Added new /FHIR/R4 base path

### PR DESCRIPTION
**Pull request by TomSmithCoding**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: proxies/live/apiproxy/resources/jsc/URLRewrite.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+ if (context.flow === 'TARGET_REQ_FLOW') {
+  var path = context.getVariable('proxy.pathsuffix')
+  print("proxy.pathsuffix is: " + path)
```
Comment (alexhawdon) [inline proxies/live/apiproxy/resources/jsc/URLRewrite.js:3]: remove

File: proxies/sandbox/apiproxy/resources/jsc/URLRewrite.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+ if (context.flow === 'TARGET_REQ_FLOW') {
+  var path = context.getVariable('proxy.pathsuffix')
+  print("proxy.pathsuffix is: " + path)
```
Comment (alexhawdon) [inline proxies/sandbox/apiproxy/resources/jsc/URLRewrite.js:3]: Remove

File: proxies/sandbox/apiproxy/resources/jsc/URLRewrite.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  {
+   fixedpath = path.slice(9);   //Length of /FHIR/R4
+  }
+  else
+  {
+     fixedpath = path; 
+  }
+  
+  context.setVariable('fixedpath', fixedpath)
+  //queryParams = context.getVariable('request.querystring')
```
Comment (alexhawdon) [inline proxies/sandbox/apiproxy/resources/jsc/URLRewrite.js:15]: Remove

File: tests/user_restricted/test_user_restricted.py [hunk 4]
```diff
@@ -585,3 +587,69 @@ def test_update_patient_with_missing_x_request_header(self, headers_with_token):
         helpers.check_retrieve_response_body(update_response, update[6]["response"])
         helpers.check_response_status_code(update_response, 412)
         helpers.check_response_headers(update_response, self.headers)
+
+        # Light weighted tests to ensure we cah hit the old end points
```
Comment (alexhawdon) [inline tests/user_restricted/test_user_restricted.py:591]: Convert to docstring

**Review by alexhawdon — CHANGES_REQUESTED** [review]

File: tests/user_restricted/test_user_restricted.py [hunk 5]
```diff
@@ -585,3 +587,69 @@ def test_update_patient_with_missing_x_request_header(self, headers_with_token):
         helpers.check_retrieve_response_body(update_response, update[6]["response"])
         helpers.check_response_status_code(update_response, 412)
         helpers.check_response_headers(update_response, self.headers)
+
+        """Light weighted tests to ensure we cah hit the old end points"""
```
Comment (alexhawdon) [inline tests/user_restricted/test_user_restricted.py:591]: No not here - should be directly under the Class/Function definition

**Review by alexhawdon — CHANGES_REQUESTED** [review]

**Review by alexhawdon — APPROVED** [review]

**Review by arvinkumarthoom — APPROVED** [review]
Reviewed

**Pull request merged**